easybuild/easyconfigs/s/SuperLU/SuperLU-5.1.1-foss-2016a.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+
+name = 'SuperLU'
+version = '5.1.1'
+
+homepage = 'http://crd-legacy.lbl.gov/~xiaoye/SuperLU/'
+description = """SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems of linear equations on high performance machines."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'opt': True, 'pic': True}
+
+source_urls = ['http://crd-legacy.lbl.gov/~xiaoye/SuperLU/']
+sources = ["superlu_%(version)s.tar.gz"]
+
+# Let's store the checksum in order to be sure it doesn't suddenly change
+checksums = ['260a3cd90b2100122abff38587a8290a']
+
+# Set to ON to generate shared library
+configopts = "-DBUILD_SHARED_LIBS=OFF"
```

Comment:
Well, ok...

EB has a way of dealing with that BTW:

```python
configopts = [
    "-DBUILD_SHARED_LIBS=OFF",
    "-DBUILD_SHARED_LIBS=ON",
]
```

but you'd to adjust more below (the `+=` lines for `configopts`, and the `installopts` which is now specific to static libs

but fine, OK to leave as is I guess